supabase/migrations/20260723144749_dedicated_native_builder.sql

```
@@ -0,0 +1,145 @@
+-- Dedicated native builders: org-scoped request + provisioning state.
+-- Actual runner routing lives in the external builder service; Capgo stores
+-- preference, fallback policy, and customer-facing status.
+
+ALTER TABLE "public"."build_requests"
+  ADD COLUMN IF NOT EXISTS "builder_pool" text;
+
+COMMENT ON COLUMN "public"."build_requests"."builder_pool" IS
+  'Preferred builder pool at request time (dedicated or shared). May differ from the pool that actually ran the job when shared fallback is enabled. Null for legacy rows.';
```

SQLFluff (4.2.2)

[error] Line is too long (172 > 80).

(LT05)